src/main/kotlin/io/bazel/kotlin/builder/KotlinBuilderComponent.java

```diff
@@ -38,10 +38,10 @@
 
     Kotlin2JsTaskExecutor jsTaskExecutor();
 
-    BazelWorker worker();
+    CompileKotlin compileKotlin();
```

Comment:
The name suggests to me an action "Compile Kotlin" but I think this is producing a Kotlin Worker Request Handler or something? Could it be named as such?

Reply:
For example the jdeps merger equivalent is `work()` - perhaps there's a naming you can find that is consistent between them and descriptive about the thing being produced?

Reply:
Sure -- naming is the hardest part.

I would say that it is compiling kotlin (for the most part) and the request handling is an implementation detail. But I'm not married to that.